src/components/Tearsheet/Tearsheet.js

```diff
@@ -141,6 +147,7 @@ class Tearsheet extends Component {
                         kind="ghost-danger"
                         onClick={onDeleteButtonClick}
                         renderIcon={icon}
+                        tabIndex={this.state.loading ? -1 : 0}
```

Comment:
This feels like an opportunity for us to pull this out as a variable and prevent us from repeating the ternary.